react/src/hooks/useActiveFeatureFlags.ts

```diff
@@ -8,7 +9,7 @@ export function useActiveFeatureFlags(): string[] | undefined {
     // would be nice to have a default value above however it's not possible due
     // to a hydration error when using nextjs
 
-    useEffect(() => {
```

Comment:
I can prevent a "flickering" ui where an initial render is not displaying an element, and a second render caused `setState` does display an element.

`useEffect` looks like: 
1. render 
1. paint browser 
1. run effect 
1. mutate state 
1. re-render 
1. paint browser

`useLayoutEffect` looks like: 
1. render 
1. run layoutEffect
1. mutate state
1. re-render
1. paint browser

Downside is that useLayoutEffect can have negative impact on render performance since it's blocking whereas useEffect is asynchronous.

Reply:
Oh, I see, thanks!